local-cli/cli.js

```diff
@@ -8,163 +8,19 @@
  */
 'use strict';
 
+// This file must be able to run in node 0.12 without babel so we can show that
```

Comment:
0.12 is right here, The file needs to work in node 0.12 without babel so it can show the error message that says that we support only node 4+ instead of a syntax error.